test/jdk/jdk/jfr/event/gc/detailed/TestGCLockerEvent.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Alibaba Group Holding Limited. All Rights Reserved.
```

Comment:
Would it be possible to keep with the general format of copyright messages in other code, i.e. "Copyright (c) <year>, <copyright holder>. ..."? I.e. if possible please add a comma after the year.

Reply:
Fixed.